atom/app/atom_main.cc

```diff
@@ -69,11 +69,11 @@ void FixStdioStreams() {
   // For details see https://github.com/libuv/libuv/issues/2062
   struct stat st;
   if (fstat(STDIN_FILENO, &st) < 0 && errno == EBADF)
-    freopen("/dev/null", "r", stdin);
+    ignore_result(freopen("/dev/null", "r", stdin));
```

Comment:
I think the better fix here would be `CHECK(freopen(...))`. On success, the return value is just `stdin`, but on failure the return value is `null`.

Reply:
I see this is already merged and backported, but we should at least switch this to `CHECK` on master.